netjsonconfig/backends/raspbian/schema.py

```diff
@@ -0,0 +1,4 @@
+from ...schema import schema as default_schema
+from ...utils import merge_config
+
+schema = merge_config(default_schema, {})
```

Comment:
@ritwickdsouza at least the minimum required stuff. This part will become clearer when we'll be able to start viewing the configuration form in web UI